streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/BaseRepartitionNode.java

```diff
@@ -30,7 +30,6 @@
     protected final String repartitionTopic;
     protected final ProcessorParameters processorParameters;
 
-
```

Comment:
Very fair point, totally agree. Sorry about that. There're not many changes related to "cleanup" in this PR, but if you think it creates noise and makes it harder to do the review, I'll remove all cleanup related code.

Reply:
Thanks for your understanding. I just mentioned it because it seemed like there wasn't a ton of review activity. Just thought I'd share the tactic with you, since rightly or wrongly, the large "diff" numbers in the PR can scare off reviewers.

For myself, I didn't have trouble overlooking it.

Reply:
Makes total sense, thanks for sharing. I will definitely take this into account for the future PRs.